src/planwise/client/projects2/components/settings.cljs

```diff
@@ -85,6 +85,10 @@
                     :on-change #(reset! value (-> % .-target .-value))
                     :value @value}])))
 
+(defn- count-sites
+  [tags {{:keys [total filtered]} :dataset-sites}]
+  [:p "Available sites: " (if (empty? tags) total filtered) " / " total])
```

Comment:
The label here would be "Usable sites" or "Selected sites". It should also display regardless of whether tags have been selected. Ie. with no tags, all sites within the region will be used.